apps/webapp/src/components/shared/toast-content.tsx

```diff
@@ -14,6 +14,20 @@ export const loadingTxToast: ToastFnProps = {
   duration: Infinity, // Building could potentially take a long time. Expected to be dismissed manually.
 };
 
+export const broadcastingTxToast = (): ToastFnProps => ({
+  main: 'Broadcasting transaction',
+  variant: 'default',
```

Comment:
Soon we'll have design resources to help us think through and polish the flow of this a bit more. I suspect that toasts aren't the way to go with these given how quickly they are dismissed between stages. A table this is probably better:

```
[x] Authorizing
[-] Building
[ ] Broadcasting
[ ] Waiting for confirmation
```

But this is for a later discussion when the design team is onboarded.

Reply:
toasts provide an "update" function that could be used to feed status into a sustained toast, or we could have some other ui. i think personally i'd like to see it tracked in the header